### PR TITLE
Da/terminal update

### PR DESCRIPTION
**Pull request by tadad**

### CODE REVIEW

File: src/chess_app/chess_engine.md [hunk 1]
```diff
@@ -131,9 +131,8 @@ Below, you'll find the full code for the CLI version of the app.
 You can build it and install it on a node using `kit`.
 You can interact with it in the terminal, primitively, like so (assuming your first node is `fake.os` and second is `fake2.os`):
 ```
-/a our@my_chess:my_chess:template.os
-/m {"NewGame": {"white": "fake.os", "black": "fake2.os"}}
-/m {"Move": {"game_id": "fake2.os", "move_str": "e2e4"}}
+m our@my_chess:my_chess:template.os {"NewGame": {"white": "fake.os", "black": "fake2.os"}}
+m our@my_chess:my_chess:template.os {"Move": {"game_id": "fake2.os", "move_str": "e2e4"}}
```
Comment (dr-frmr) [inline src/chess_app/chess_engine.md:135]: don't these JSON objects need single-quotes surrounding?

File: src/cookbook/file_transfer.md [hunk 2]
```diff
@@ -1189,7 +1189,7 @@ There you have it!
 Try and run it, you can download a file with the command
 
 ```
-/m our@file_transfer:file_transfer:template.os {"Download": {"name": "dawg.jpeg", "target": "fake2.os@file_transfer:file_transfer:template.os"}}
+m our@file_transfer:file_transfer:template.os {"Download": {"name": "dawg.jpeg", "target": "fake2.os@file_transfer:file_transfer:template.os"}}
```
Comment (dr-frmr) [inline src/cookbook/file_transfer.md:1192]: single quotes?

Comment (tadad) [reply src/cookbook/file_transfer.md]: yes, just fixed all instances of this

**Review by dr-frmr — APPROVED** [review]

**Review by hosted-fornet — COMMENTED** [review]
PR looks good, nice work :+1: 

I have two additional requests that I've opened issues to track:

1. We should document how piping works & show a few examples, #94 
2. We should document how to write & deploy scripts, #95

**Pull request merged**